### DISCUSSION TITLE
Lesson 5: test_simple_storage.py error message rolodex

### DISCUSSION THREAD

**Discussion by ArcusConsulting** [smartcontractkit/full-blockchain-solidity-course-py#235] [Q&A] [answered]
At the ~4:50 mark, I successfully ran the test_deploy() function with no errors:

![image](https://user-images.githubusercontent.com/91853555/138198053-6f0850c8-6fbf-4cd2-b438-f6b2f2591728.png)

However, when I run the test_updating_storage() function, it looks like my 2 tests run successfully, but it when goes to terminate the RPC client I get a slew of error messages:

![image](https://user-images.githubusercontent.com/91853555/138198097-61142556-2634-40cb-aa6c-effec20dc573.png)

**Comment by ArcusConsulting** [comment]
The only thing that I can read that makes sense is "Web3 is not connected"... But if that's the case, then my first function should have returned similar errors, no?

**Comment by ArcusConsulting** [comment]
![image](https://user-images.githubusercontent.com/91853555/138198368-478d59f4-9c04-411a-8134-72d754f8cfca.png)

**Comment by PatrickAlphaC** [comment, marked as answer]
This is a common bug. You need to `wait` whatever your last transaction is.

In this case, do:
```python
tx = simple_storage.store(expected, {"from": account})
tx.wait(1)
```

And you'll be all set!

**Reply by ArcusConsulting** [reply]
The people's champ... Thank you!

**Reply by ArcusConsulting** [reply]
Just tested this and it worked. Moving on!
